src/tutorials/konnector/2fa.md

```diff
@@ -0,0 +1,64 @@
+You have successfully developped your connector but you realize that some accounts need two factor
+authentication (2FA) to work.
+
+Here is how you can add this to your connector
+
+## 2FA detection
+
+Every target website being different, you will have to develop it yourself. Some websites return a
+specific url, some others display a special html content. Take a look at [the github connector](https://github.com/konnectors/github) for
+real life example.
+
+## Home Automatic Successul Login deactivation
+
+By default, the "Home" application will suppose the connector's login is a success after 8s, which is bad when
```

Comment:
I still don't understand why we are doing this instead of modifying most connectors and let the few that need a such timeout (like the oauth ones) opt for it manually like `lib.automaticLoginSuccessAfter(8000)`

Reply:
Because that would mean changing most of the connecteur and the home application at the same time and I don't think we want to do that

Reply:
Let me disagree for the first part ;-) I really do prefer that we add an opt-in in specific connectors than an opt-out in most of them